### PR TITLE
Dynamically change the remaining number of characters required.

### PR DESCRIPTION
**Pull request by aymenhamada**
Dynamically change the remaining number of characters required.

Issue : #81

### CODE REVIEW

File: src/views/pages/ApplicationsView/ApplicationSteps.js [hunk 1]
```diff
@@ -486,7 +485,7 @@ function FormChallenge({
         validators={['required', 'isNotShort']}
         errorMessages={[
           'This is a required field',
-          'Text too short. Put in atleast 150 chars.'
+          `Text too short. Put in atleast 150 chars. ${formData.q1 && formData.q1.length ? "Remaining " + (150 - formData.q1.length) + `${formData.q1.length < 149 ?  " characters" : " character"}`: ""} `
```
Comment (Abhishek-kumar09) [inline src/views/pages/ApplicationsView/ApplicationSteps.js:488]: You should only display:
Text too short. Put in `x` characters more.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Nicely Done.
Run `npm run format` before pushing your next commit.

File: src/views/pages/ApplicationsView/ApplicationSteps.js (hunk 1 above)
Comment (aymenhamada) [reply src/views/pages/ApplicationsView/ApplicationSteps.js:488]: Sure i'll fix that.

**Comment by aymenhamada** [comment]
Added a variable containing the minimum number of characters required
`  const MIN_CHAR = 150;`

Also added a function to calculate the real length of a string (Removing white space, so user can't submit 150 wihte space)
`const countChar = string => {
    return string.replace(/\s/g, '').length;
 };`

**Review by Abhishek-kumar09 — APPROVED** [review]

**Comment by KeenWarrior** [comment]
Nice work @aymenhamada  🥳

**Pull request merged**